modules/crm_core_default_matching_engine/config/install/crm_core_match.matcher.household.yml

```diff
@@ -0,0 +1,9 @@
+langcode: en
```

Comment:
Default configuration is used by Collect (demo), not sure either does it need to be removed. 
Dropped the crm_core_mdefault_matching_engine module. 
All useful code was moved to crm_core_match (including ReadMe.txt which I appended to the crm_core_match read me file.